packages/shared/src/hooks/search/useSearchSuggestions.ts

```diff
@@ -18,7 +19,7 @@ export const useSearchSuggestions: UseSearchSuggestions = (args) => {
   const { data, isLoading } = useQuery(
     generateQueryKey(RequestKey.SearchHistory, user),
     getSearchSuggestions,
-    { ...disabledRefetch, enabled: !!user },
+    { ...disabledRefetch, enabled: !args.disabled && !!user },
```

Comment:
Since it's optional we should say `!args?.disabled` right?

Reply:
I think `args.disabled` is optional, but `args` itself is not? https://github.com/dailydotdev/apps/blob/b94b97299a225854f6f11608cdb01da72f9c9412/packages/shared/src/hooks/search/useSearchSuggestions.ts#L9-L15